faust_large_message_serializer/clients/storing_client.py

```diff
@@ -0,0 +1,57 @@
+from typing import Union
+from uuid import uuid4
+
+from loguru import logger
+
+from faust_large_message_serializer.config import LargeMessageSerializerConfig
+
+
+class StoringClient:
+
+    VALUE_PREFIX = "values"
+    KEY_PREFIX = "keys"
+    IS_BACKED = b"\x01"
+    IS_NOT_BACKED = b"\x00"
+
+    def __init__(
+        self,
+        config: LargeMessageSerializerConfig
+    ):
+        self._config = config
```

Comment:
I would make base_path and max_size parameters and fields to make it independent of the config